src/lib.rs

```diff
@@ -22,3 +22,20 @@ pub mod program;
 pub mod screen;
 pub mod sort;
 pub mod ui;
+
+use shellexpand::full;
+use std::path::PathBuf;
+
+/// Converts the provided path by user to a path that can be glob'd
+/// Directories are changed from /home/etc to /home/etc/*
+pub fn convert_to_globable(path: &str) -> Result<String, String> {
+    let expanded_path = full(path).map_err(|e| format!("\"{}\": {}", e.var_name, e.cause))?;
+    // remove escaped spaces
+    let absolute_path = String::from(expanded_path).replace(r"\ ", " ");
+    // If path is a dir, add /* to glob
+    let mut pathbuf = PathBuf::from(&absolute_path);
+    if pathbuf.is_dir() {
+        pathbuf = pathbuf.join("*");
+    }
+    Ok(pathbuf.to_string_lossy().to_string())
```

Comment:
```rust
      Ok(pathbuf.to_string_lossy().to_string())
```

Would this mean that we only handle valid UTF-8 paths?
Seems that [glob can't accept invalid utf-8 as a pattern anyway.](
https://github.com/rust-lang-nursery/glob/issues/78)


Because the following path would not work.

python3
```python
invalid_utf8_file = b"\xc3\x28"
with open(invalid_utf8_file, "w+") as f:
    f.write("invalid\n")
```